### ISSUE TITLE
Snapchat app crash when sending sticker only on TestFlight / Live App

### ISSUE THREAD

**Issue by marcnpopa** [Snapchat/creative-kit-sample#38]
Hi, 

Our iOS app fails to send a sticker, by crashing the Snapchat app. This happens only on TestFlight builds. 
When building directly on the phone with any build configuration it works well. The sticker share was working fine with the same code more than two weeks ago.

We use snap-kit’s SCSDKCreativeKit with Carthage to send stickers. (From binary "https://raw.githubusercontent.com/Snapchat/snap-kit-carthage/repo/SCSDKCreativeKit.json" )"

Thank you in advance for any leads on how to solve this issue,
Marc

*
Code:
```
private lazy var snapAPI = SCSDKSnapAPI()
  
    func shareSnap(stickerImage: UIImage,
                         attachmentUrl: String? = "https://itunes.apple.com/us/app/pop-group-chat/idXXXXXXXXXX?mt=8",
                         context: String) {
      let photo = SCSDKSnapPhoto(image: stickerImage)
      let content = SCSDKPhotoSnapContent(snapPhoto: photo)
      content.attachmentUrl = attachmentUrl
      snapAPI.startSending(content) { error in
          if let error = error {
              log.error("Error sharing to snap: \(error.localizedDescription)")
              return
          }
       AnalyticsService.logShare(context: context, method: Strings.Snapchat)
    }
 }
  
  ```

**Comment by iiikbh** [comment]
Same here .. 👎

**Comment by Khaled-BH** [comment]
Any solution ?

**Comment by marcnpopa** [comment]
Seems like Snapchat solved the issue in the 11.48 release. Just update the app and it should work!
